crates/bifrost/src/providers/replicated_loglet/sequencer/mod.rs

```diff
@@ -187,6 +228,10 @@ impl<T: TransportConnect> Sequencer<T> {
             .await
             .unwrap();
 
+        metrics::counter!(RECORDS_ENQUEUED_COUNT).increment(payloads.len() as u64);
+        metrics::counter!(RECORDS_ENQUEUED_BYTES)
+            .increment(payloads.estimated_encode_size() as u64);
```

Comment:
I'd try and be a little conservative and only add those metrics when we need them. Perhaps we should add those counters on the loglet implementation level regardless whether it's a sequencer node or not. What do you think?

Reply:
My plan was to have those metrics for the sequencer itself. Not for the loglet implementation. The point was to measure sequencer performance. 

I am okay with having other metrics on the loglet level as well.

Reply:
After a second thought, I guess that makes more sense.